tests/jerry/fail/regression-test-issue-5085.js

```diff
@@ -0,0 +1,18 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+class A{}
```

Comment:
Do we need this line?

Reply:
Yes, without it an assertion fails before the bug would cause a segmentation fault:
```
ICE: Assertion 'context_p->scope_stack_size == PARSER_MAXIMUM_DEPTH_OF_SCOPE_STACK' failed at jerry-core/parser/js/js-parser-expr.c(parser_parse_class):1068
Error: JERRY_FATAL_FAILED_ASSERTION
```

Reply:
Oh I was mistaken, I was looking at a different branch.

With this patch applied this assertion does not fail with that line removed and the correct syntax error is raised.